frontend/src/tests/ForestClientInput.spec.ts

```diff
@@ -102,4 +121,286 @@ describe('ForestClientInput', () => {
+    //     const inputField: HTMLInputElement = input.element as HTMLInputElement;
+    //     await input.setValue(TEST_SUCCESS_FOREST_CLIENT_NUMBER);
+    //     expect(inputField.value).toBe(TEST_SUCCESS_FOREST_CLIENT_NUMBER);
+    //     expect(isLoading()).toBe(false);
+
+    //     await wrapper.find("[aria-label='Add Client Numbers']").trigger('click');
+    //     await flushPromises();
+
+    //     expect(isLoading()).toBe(true);
+    //     expect(wrapper.find("[aria-label='Add Client Numbers']").element.textContent).contains('Loading')
+    // });
+
+    it('should test the delete forest client number case', async () => {
+        const multipleSuccessInputs = `${TEST_SUCCESS_FOREST_CLIENT_NUMBER},${TEST_SUCCESS_FOREST_CLIENT_NUMBER_2}`;
+        const input = wrapper.find('#forestClientInput');
+        const inputField: HTMLInputElement = input.element as HTMLInputElement;
+        await input.setValue(multipleSuccessInputs);
+        expect(inputField.value).toBe(multipleSuccessInputs);
+
+        await wrapper.find("[aria-label='Add Client Numbers']").trigger('click');
+        await flushPromises();
+
+        const setVerifiedForestClients = wrapper.emitted(
+            'setVerifiedForestClients'
+        );
+
+        expect(wrapper.emitted()).toHaveProperty('setVerifiedForestClients');
+        expect(setVerifiedForestClients).toHaveLength(2);
+        expect(
+            setVerifiedForestClients
+                ? setVerifiedForestClients[0][0]
+                : undefined
+        ).toEqual(TEST_SUCCESS_FOREST_CLIENT_NUMBER);
+        expect(
+            setVerifiedForestClients
+                ? setVerifiedForestClients[1][0]
+                : undefined
+        ).toEqual(TEST_SUCCESS_FOREST_CLIENT_NUMBER_2);
+        // the forest client information card shows and verify the information
+        expect(wrapper.html().includes('Verified Client ID information')).toBe(
+            true
+        );
+        expect(
+            wrapper
+                .find('.client-id-wrapper')
+                .text()
+                .includes(TEST_SUCCESS_FOREST_CLIENT_NUMBER)
+        ).toBe(true);
+        expect(
+            wrapper
+                .find('.org-name-wrapper')
+                .text()
+                .includes(TEST_FOREST_CLIENT_NAME)
+        ).toBe(true);
+        expect(
+            wrapper.find('.org-status-wrapper').text().includes('Active')
+        ).toBe(true);
+        // the forest client input should now be empty
+        expect(inputField.value).toBe('');
+
+        const deleteButton = wrapper.find('.custom-carbon-icon--trash-can');
+        deleteButton.trigger('click');
+        await flushPromises();
+        expect(
+            wrapper
+                .find('.client-id-wrapper')
+                .text()
+                .includes(TEST_SUCCESS_FOREST_CLIENT_NUMBER)
+        ).toBe(false);
+    });
+
+    it('should cleanup error when property change', async () => {
```

Comment:
I think this test case is about when input change, the error will be cleanup

Reply:
And we need to add another test case, when userId or roleId changed, the forest client errors and card will be removed, and the resetVerifiedForestClients emit method should be called